RiotSwiftUI/Modules/Authentication/Common/AuthenticationModels.swift

```diff
@@ -23,22 +23,40 @@ enum AuthenticationFlow {
 }
 
 /// Errors that can be thrown from `AuthenticationService`.
-enum AuthenticationError: String, Error {
+enum AuthenticationError: String, LocalizedError {
     /// A failure to convert a struct into a dictionary.
     case dictionaryError
     case invalidHomeserver
     case loginFlowNotCalled
-    case missingRegistrationWizard
     case missingMXRestClient
+    
+    var errorDescription: String? {
+        switch self {
+        case .invalidHomeserver:
+            return VectorL10n.authenticationServerSelectionGenericError
+        default:
+            return VectorL10n.errorCommonMessage
```

Comment:
When we log these errors in rageshakes, how do we print the errors? Just want to make sure we do not loose context of the original error.

Reply:
I've made sure all errors are logged before throwing, so these user-facing messages should be fine now.